lib/src/builder_functions.dart

```diff
@@ -1,9 +1,9 @@
 import 'package:flutter/widgets.dart';
 
 typedef RxBuilder<T> = Widget Function(BuildContext context, T data);
-typedef ErrorBuilder<T> = Widget Function(BuildContext context, Object error);
+typedef ErrorBuilder<T> = Widget Function(BuildContext context, Object? error);
```

Comment:
Why is error nullable here? Normally the ErrorBuilder should never be called if there is no error or do I miss something here?

Reply:
You are right, didn't pay attention to this fact. Will correct.